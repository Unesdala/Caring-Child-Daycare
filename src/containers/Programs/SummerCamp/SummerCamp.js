import React, { Component } from 'react';
import commonUtils from '../../../components/lib/commonUtils';
import DefaultSummerCampContent from './SummerCampContent';
import DefaultFooter from '../../../components/Footer/Footer';

export class SummerCamp extends Component {
  constructor(props) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount() { this.commonUtils.setTitleAndScroll('Twos'); }

  render() {
    return (
      <div className="page-content">
        <DefaultSummerCampContent />
        <DefaultFooter />
      </div>
    );
  }
}

export default SummerCamp;
