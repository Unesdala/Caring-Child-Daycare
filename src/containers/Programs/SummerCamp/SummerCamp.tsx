import React, { Component } from 'react';
import CommonUtils from '../../../lib/commonUtils';
import DefaultSummerCampContent from './SummerCampContent';
import DefaultFooter from '../../../components/Footer/Footer';

export class SummerCamp extends Component {
  public commonUtils: typeof CommonUtils = CommonUtils;

  componentDidMount(): void { this.commonUtils.setTitleAndScroll('Summer Camp', window.screen.width); }

  render(): JSX.Element {
    return (
      <>
        <div className="page-content">
          <DefaultSummerCampContent />
        </div>
        <DefaultFooter />
      </>
    );
  }
}

export default SummerCamp;
