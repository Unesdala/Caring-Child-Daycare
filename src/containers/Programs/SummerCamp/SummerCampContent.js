import React from 'react';

const SummerCampContent = () => (
  <main className="summer-camp">
    <p />
  </main>
);

export default SummerCampContent;
