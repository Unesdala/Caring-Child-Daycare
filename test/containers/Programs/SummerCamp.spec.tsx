/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { SummerCamp } from 'src/containers/Programs/SummerCamp/SummerCamp';
import SummerCampContent from 'src/containers/Programs/SummerCamp/SummerCampContent';

describe('Summer Camp', () => {
  const wrapper = shallow(<SummerCamp />);
  it('renders', () => {
    expect(wrapper.find(SummerCampContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(SummerCampContent).dive().find('main.summer-camp-main--content').exists()).toBe(true);
  });
  
});
