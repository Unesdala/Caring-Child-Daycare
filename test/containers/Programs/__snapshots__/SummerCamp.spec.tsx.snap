// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Summer Camp renders 1`] = `ShallowWrapper {}`;
